GitUI/GitExtensionsForm.cs

```diff
@@ -303,20 +302,6 @@ private void RestorePosition(string name)
             }
         }
 
-        public int GetCurrentDeviceDpi()
-        {
-#if TARGETING_DOTNET47
-            int deviceDpi = DeviceDpi;
```

Comment:
Hi! Why remove this? You will use system DPI instead of current monitor DPI.

Reply:
Hi,

I removed it because I couldn't find any reference to symbol `TARGETING_DOTNET47`, and concluded this code could not have any effect. Am I missing something?

I agree that per-monitor DPI awareness is not something we handle yet. Doing so properly will require appropriate refreshing and considerably more testing. I've added #4822.

Reply:
I've used it internally for testing with .Net 4.7 and per-monitor DPI

At least it should be a method to get current DPI that get `Control` as a parameter.

BTW we don't need `DpiY`, because of it always equal to `DpiX`.